genequery-rest/src/test/kotlin/gq/rest/api/GeneSetEnrichmentControllerTest.kt

```diff
@@ -27,16 +24,6 @@ import org.springframework.web.context.WebApplicationContext
 import java.nio.charset.Charset
 
 
-class PatchEnvPropsAppCtxInitializer : ApplicationContextInitializer<ConfigurableApplicationContext> {
```

Comment:
Make sure you're up-to-date with current version of the code base. This class must exist.